src/main/java/com/cleanroommc/groovyscript/compat/mods/additionalenchantedminer/WorkBenchPlus.java

```diff
@@ -0,0 +1,95 @@
+package com.cleanroommc.groovyscript.compat.mods.additionalenchantedminer;
+
+import com.cleanroommc.groovyscript.api.GroovyLog;
+import com.cleanroommc.groovyscript.api.documentation.annotations.*;
+import com.cleanroommc.groovyscript.compat.mods.ModSupport;
+import com.cleanroommc.groovyscript.helper.recipe.AbstractRecipeBuilder;
+import com.cleanroommc.groovyscript.registry.VirtualizedRegistry;
+import com.yogpc.qp.recipe.WorkbenchRecipe;
+import com.yogpc.qp.tile.ItemDamage;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+import org.jetbrains.annotations.Nullable;
+import scala.collection.JavaConversions;
+import scala.collection.Map;
+
+@RegistryDescription
+public class WorkBenchPlus extends VirtualizedRegistry<WorkbenchPlusRecipe> {
+
+    @Override
+    public void onReload() {
+        removeScripted().forEach(recipe -> WorkbenchRecipe.removeRecipe(recipe.getLocation()));
+        restoreFromBackup().forEach(ModSupport.ADDITIONAL_ENCHANTED_MINER.get().WorkBenchPlus::add);
```

Comment:
calling this method adds the method to the scripted list, which should not be done.